### DISCUSSION TITLE
buddhism/2023/sanskrit/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#108] [General]
# buddhism/2023/sanskrit/

http://shanenull.com/buddhism/2023/sanskrit/

**Comment by shane0** [comment]
http://shanenull.com/buddhism/2024/quotes/
anything changing is an illusion and not real

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2024/evening_ritual/

**Comment by shane0** [comment]
# three seals

- https://shanenull.com/buddhism/2023/sanskrit/#three-dharma-seals
- three seals comments https://youtu.be/CZcMFObyxdA

**Reply by shane0** [reply]
https://zh.wikipedia.org/zh-hant/%E4%B8%89%E6%B3%95%E5%8D%B0
